src/coverlet.console/Program.cs

```diff
@@ -223,7 +225,12 @@ static int Main(string[] args)
                         exceptionMessageBuilder.AppendLine($"The {dThresholdStat.ToString().ToLower()} method coverage is below the specified {dThreshold}");
                     }
 
-                    throw new Exception(exceptionMessageBuilder.ToString());
+                    if (dThresholdAct == ThresholdAction.Fail)
```

Comment:
I prefer another `else if (dThresholdAct == ThresholdAction.Warning)` or a switch, today we've 2 condition and should not change...but I want to be sure that semantic won't change if we add other state in future.

Reply:
changed to using a switch statement